src/main/java/org/broadinstitute/barclay/argparser/PositionalArgumentDefinition.java

```diff
@@ -110,6 +110,48 @@ public void validateValues(final CommandLineArgumentParser commandLineArgumentPa
         }
     }
 
+    @Override
+    public String getDocString() { return positionalArgumentsAnnotation.doc(); }
+
+    /**
+     * Return a string with the usage statement for this positional argument.
+     * @param argumentColumnWidth width reserved for argument name column display
+     * @param descriptionColumnWidth width reserved for argument description column display
+     * @return the usage string for this argument
+     */
+    public String getArgumentUsage(final int argumentColumnWidth, final int descriptionColumnWidth) {
+
+        final StringBuilder sb = new StringBuilder();
+        sb.append("--").append("POSITIONAL (must be first)");
```

Comment:
@lbergelson There can only be one "positional" option per CLP. It can be a collection, so it can have multiple values, but they have to be first in the list. Its basically just an arg without a name. We should probably just get rid of it, but GATK and Picard each have one, I think.

Reply:
Huh, I thought we supported multiple positionals, sorry, ignore my comment then.